src/devices/third_reality.ts

```diff
@@ -282,13 +282,13 @@ export const definitions: DefinitionWithExtend[] = [
         zigbeeModel: ["3RSB02015Z"],
         model: "3RSB02015Z",
         vendor: "Third Reality",
-        description: "Smart blind Gen2",
+        description: "Third Reality TRZB3",
```

Comment:
```suggestion
        description: "TRZB3",
```

Thrid reality is alrady in the `vendor`.

Reply:
ok